docs/file-based-configuration.md

```diff
@@ -223,7 +223,24 @@ resource:
       operatingsystem: # Detects OS-level attributes (os.*)
       process:         # Detects process-level attributes (process.*)
       processruntime:  # Detects process runtime attributes (process.runtime.*)
-```  
+```
+
+### Plugins Configuration
+
+For more details and updates about Plugins Configuration, see:
+[Plugins documentation](plugins.md)
+
+``` yaml
+plugins/development:
```

Comment:
Shouldn't it go under insturmentatio/development.dotnet.plugin?
Is there any consensus about putting it under semantic-confuguration?

Reply:
Lets keep it here, at least for now.